indico/modules/events/papers/lists.py

```diff
@@ -176,6 +193,9 @@ def __init__(self, event, user):
             'items': ('state',),
             'filters': {'items': {}}
         }
+        judging_unassigned_choices = {role.value: role.title for role in PaperReviewingRole
+                                      if role is not PaperReviewingRole.judge}
+        self.static_items['unassigned']['filter_choices'] = OrderedDict(judging_unassigned_choices.items())
```

Comment:
Adding something to an OrderedDict only makes sense when it's sorted before or comes from a sorted collection - but neither applies to a dict ;)

So you'll have to use `sorted()` there or you can simply use `judging_unassigned_choices` directly (but having a proper order would be better)